scripts/cnode-helper-scripts/cncli.sh

```diff
@@ -127,11 +127,18 @@ getLedgerData() { # getNodeMetrics expected to have been already run
 }
 
 getConsensus() {
-  getProtocolParams
+  if [[ -n ${subarg} ]] && [[ ${subarg} != "all" ]]; then
+    getProtocolParamsHist "${subarg}"
+  elif [[ ${subarg} == "all" ]]; then
+    getProtocolParamsHist "${EPOCH}"
+  else
+    getProtocolParams
+  fi
```

Comment:
1. subarg can contain any number of values, its general purpose for all subcommands to cncli. Ie it can be 'force' for leaderlog etc that also call this getConsensus function. 
Please change getConsensus function to look at input params to function and pass epoch number to it if a historical epoch is requested. 
2. This was a problem since before, but if getProtocolParamsHist or getProtocolParams returns an error code, this function should also return with an error no further processing being done.

Suggested change (including code change for all places calling this function):
```
if isNumber "$1"; then
  getProtocolParamsHist "$1" || return 1
else
  getProtocolParams || return 1
fi
```

Reply:
getConsensus has been updated as:

>getConsensus() {
  if isNumber "$1" ; then
    getProtocolParamsHist "$1" || return 1
  elif [[ ${subarg} == "all" ]]; then
    getProtocolParamsHist "${EPOCH}" || return 1
  else
    getProtocolParams || return 1
  fi